game/hud/src/components/HUDFullScreen.tsx

```diff
@@ -190,11 +204,15 @@ class HUDFullScreen extends React.Component<FullScreenNavProps, FullScreenNavSta
   }
 
   private renderCharacter = (prop: { active: boolean }) => {
-    return <Character visibleComponent={this.state.visibleComponent} />;
+    return <Character visibleComponent={this.state.visibleComponent}/>;
```

Comment:
Are you manually removing this white-space or using a plugin? The white-space here should be here as it was.

Reply:
this was my auto format settings, I have resolved this I believe.